src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -24,7 +25,7 @@ public static HtmlEncoder Create(params UnicodeRange[] allowedRanges)
         }
     }
 
-    public sealed class DefaultHtmlEncoder : HtmlEncoder
+    internal sealed class DefaultHtmlEncoder : HtmlEncoder
```

Comment:
New namespace does have a conceptual cost too. I don't think anybody would mind leaving the types public (int he main namespace), if there was a clear benefit of doing so that we could articulate. In the API review meeting I talked about us two having a discussion about it, but I could not come up with a single reason/example for why we would leave the types public.